experiments/MERA/modified_binary_mera_lib.py

```diff
@@ -0,0 +1,535 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+""" 
+modified binary MERA optimization
+parts of the following code are based on code written by Glen Evenbly (c) for www.tensors.net, (v1.1) 
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import sys
+sys.path.append('../../')
```

Comment:
Don't do this. Make experiments a module when possible.

Reply:
removed